variables.tf

```diff
@@ -180,3 +180,31 @@ variable "auto_scaling" {
     memory = {}
   }
 }
+##############################################################
+# Context-based restriction (CBR)
+##############################################################
+
+variable "cbr_rules" {
+  type = list(object({
+    description = string
+    account_id  = string
+    rule_contexts = list(object({
+      attributes = optional(list(object({
+        name  = string
+        value = string
+    }))) }))
+    enforcement_mode = string
+    tags = optional(list(object({
+      name  = string
+      value = string
+    })))
+    operations = optional(list(object({
```

Comment:
There is no option for operations so hardcode It to the default value you have above